tests/backup/backup_share_test.go

```diff
@@ -3418,7 +3418,8 @@ var _ = Describe("{IssueMultipleRestoresWithNamespaceAndStorageClassMapping}", f
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating new storage class %v on source cluster %s", scName, SourceClusterName))
 
 			log.InfoD("Switching cluster context to destination cluster")
-			SetDestinationKubeConfig()
+			err = SetDestinationKubeConfig()
+			log.FailOnError(err, "Failed to set destination kubeconfig")
```

Comment:
How about log.FailOnError(err, "Switching context to destination cluster failed") or log.FailOnError(err, "Failed to set destination kubeconfig") everywhere?